server/database/pool.js

```diff
@@ -0,0 +1,19 @@
+const { Pool } = require('pg');
+require('dotenv').config()
+let databaseName;
+
+if (process.env.NODE_ENV == 'test') {
+  databaseName = process.env.TEST_DATABASE_NAME;
+} else {
+  databaseName = process.env.DATABASE_NAME;
+}
+
+const pool = new Pool({
+  database: databaseName,
+  user: process.env.DATABASE_USER,
+  host: process.env.DATABASE_HOST,
+  password: process.env.DATABASE_PASSWORD,
+  port: process.env.DATABASE_PORT
+})
```

Comment:
This code does 2 things:
1 - creates a config object, mapping keys to values from the environment
2 - using that config to create a `Pool` instance.

If these things are separated, you might find it easier to create a Pool in different environments.

Reply:
Thanks Matt. I'll make sure I separate this